extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/converter/type/AbstractTypeConverter.java

```diff
@@ -121,6 +117,12 @@ public T convertFromBinding(HmDatapoint dp) throws ConverterException {
         return fromBinding(dp);
     }
 
+    protected void logTraceOrDebug(String format, Object... arguments) {
+        if (logger.isTraceEnabled()) {
```

Comment:
I thought the reason for having this if-statement is that the expressions that result in the arguments to the trace-call do not need to be computed if trace logging isn't enabled. If that is the only reason, then this helper method unfortunately defeats the purpose, since the expressions will be computed when it is called.

Reply:
Indeed, I will take care of that.